alphabase/constants/const_files/psm_reader.yaml

```diff
@@ -212,7 +212,6 @@ library_reader_base:
   reader_type: library_reader_base
   rt_unit: irt
   fixed_C57: False
-  csv_sep: "\t"
```

Comment:
We already used "on-the-fly" sep detecting, right?

Reply:
yes, this seemed to be just a leftover (wasn't used)

Reply:
I see